### PR TITLE
tests/main/security-device-cgroups-strict-enforced: extend the comments

### PR DESCRIPTION
**Pull request by bboozzoo**
Extend the comments describing what happens in the test.

### CODE REVIEW

**Review by mvo5 — APPROVED** [review]
Thank you

File: tests/main/security-device-cgroups-strict-enforced/task.yaml [hunk 1]
```diff
@@ -61,20 +61,28 @@ execute: |
...
+    # (incl. other already allowed devices)
+
     # trigger a change event for /dev/full so that we observe that the device
     # cgroup settings remain stable and we can still access /dev/zero
     snap run test-strict-cgroup.sh -c 'touch /var/snap/test-strict-cgroup/common/started; until test -e /var/snap/test-strict-cgroup/common/ready; do sleep 1; done; echo ok > /dev/zero' > run.log 2>&1 &
     retry -n 5 test -e /var/snap/test-strict-cgroup/common/started
     # save the dump
     tests.device-cgroup test-strict-cgroup.sh dump | sort > dump-before-change
-    # trigger a change of /dev/full
+    # by not the device cgroup got already set up by snap-confine when the
```
Comment (stolowski) [inline tests/main/security-device-cgroups-strict-enforced/task.yaml:75]: This comment reads weird, is something missing here?

Comment (bboozzoo) [reply tests/main/security-device-cgroups-strict-enforced/task.yaml:75]: s/not/now :smile:

Comment (stolowski) [reply tests/main/security-device-cgroups-strict-enforced/task.yaml]: Oh yes, now it makes sense :)

**Review by stolowski — APPROVED** [review]
+1

**Review by anonymouse64 — APPROVED** [review]
lgtm, thanks for this

**Pull request merged**